docs/building-images.md

```diff
@@ -120,3 +120,19 @@ as follows:
 Additionally, the following RPMs must be included in the RPMs directory as described above:
 * `elemental-register`
 * `elemental-system-agent`
+
+### SUSE Manager (SUMA)
+
+Automatic SUSE Manager registration can be configured for the image. The following items must be defined in the
+configuration file under the `suma` section:
+
+* `host` - this is the FQDN of the SUSE Manager host that the host needs to register against.
+* `activation_key` - this is the activation key that the node uses to register with.
+* `get_ssl` - this specifies whether EIB should download and install the SUMA SSL Certificate (default: false)
+
+The default SSL certificate for the SUSE Manager server can usually be found at
+https://<suma-host>/pub/RHN-ORG-TRUSTED-SSL-CERT
```

Comment:
Nit: Consider marking URLs as monospaced. But I admit that's very much a personal preference.